packages/lib/client-only/download-pdf.ts

```diff
@@ -0,0 +1,37 @@
+import type { DocumentData } from '@documenso/prisma/client';
+import { toast } from '@documenso/ui/primitives/use-toast';
+
+import { getFile } from '../universal/upload/get-file';
+
+type DownloadPDFProps = {
+  documentData: DocumentData;
+  fileName?: string;
+};
+
+export const downloadFile = async ({ documentData, fileName }: DownloadPDFProps) => {
```

Comment:
It feels strange that if this errors out the parent that calls this isn't aware

It might be more practical to just throw the error up and have the parent handle toasting and catching errors

Reply:
I'd also name this the same as the file, so `downloadFile` => `downloadPDF`